### ISSUE TITLE
Slow Xlsx export

### ISSUE THREAD

**Issue by Stepik42** [amcharts/amcharts4#2230]
If I export data in xlxs format then it will consume a lot of time. For short data interval it is fast, but if I want to export 2 months of data (aprox. 4366 records) then it will take 1,5 minutes which is pretty slow in comparison of csv export which is done within 1 second.

I'm using simple chart function:
chart.exporting.export('xlsx');

Does exist any way of improving xlxs exporting ? 
Thank you .

**Comment by martynasma** [comment]
I don't think so. We're using a 3rd party library to generate Excel files, so we don't have any control over it.

**Comment by Stepik42** [comment]
Thank's for reply.

Last thing I want to mention. When I played with data export I noticed that adapter: chart.exporting.adapter.add('data', (data, target) => {}) is called multiple times during xlsx export and always returns the same data. 
I have very similar usecase to this post: 
https://www.amcharts.com/docs/v4/tutorials/combining-local-data-from-multiple-series-for-export/

and there can be seen that during xlsx exporting the adapter is called 20+ times and always returns the same data.

This don't have to be an error necessarily but it seems strange to me.

Thanks.
